.github/workflows/pack-build_and_test.yaml

```diff
@@ -35,20 +35,24 @@ jobs:
   build_and_test:
     runs-on: ubuntu-20.04
     # When parent workflow is named "Build and Test" this shows up as:
-    #   "Build and Test / Python 3.6"
+    #   "Build and Test / Python 3.X"
     name: 'Python ${{ matrix.python-version-short }}'
     strategy:
+      fail-fast: false
+      max-parallel: 2
       matrix:
         include:
           - python-version-short: "3.6"
             python-version: 3.6.13
+          - python-version-short: "3.8"
+            python-version: 3.8.10
```

Comment:
Nice! :+1: to include 3.8 in the matrix.

@cognifloyd  Is there anything else we need to handle for this?

Reply:
There are a few repos that will also need an update I believe. I'll look at which when I'm at my computer later today.